_data/m2.yml

```diff
@@ -43,4 +43,8 @@ solution_validation:
 
 conclusions:
     section: m2_conclusions
-    title: Conclusions
\ No newline at end of file
+    title: Conclusions
+    intro: From all the answers we gathered, we found that people rely on the internet when learning something new. Usually, they start from general aspects of those subjects, and after that, they go to particular information. The most popular resources are video tutorials, online articles and courses. But before selecting what materials to learn from, they have to know if they are trustworthy. To find that, they search reviews as notes or comments or ask acquaintances.
+    pr_val: The problems people often have when trying to study is that there is a lot of information on the internet and they do not know where to start. The information is not structured on their needs, it is sometimes incomplete or taken out of the context. These facts confirm the problem discovered by us. 
+    sol_val: Moreover, when studying something new, most people do not have a certain tool they use to find materials, they have particular methods to find what they need, but they use organizational tools, to structure their tasks. Also, they do not use many extensions, but when they find out a very useful one, they stick to that.  The majority of the persons we interviewed would find very useful a tool to find suggestions about what to learn based on their needs, level of knowledge and the reviews from other users. It would be time-saving. 
```

Comment:
Here instead of "do not have a certain tool they use" I would say "do not use a certain tool" and instead of "find out a very useful one" I would say "find a very useful one". And I would integrate the last sentece into the one before, something like "The majority of the persons we interviewed would find very useful and time-saving...".

Reply:
Sunt doi useful in doua propozitii la rand.  Cred ca poti sa zici "find one that suits them, they stick to that."